src/java.desktop/share/native/common/awt/debug/debug_trace.h

```diff
@@ -62,6 +62,9 @@ JNIEXPORT void JNICALL DTrace_VPrint(const char * file, int line, int argc, cons
 JNIEXPORT void JNICALL DTrace_VPrintln(const char * file, int line, int argc, const char * fmt, va_list arglist);
 
 /* each file includes this flag indicating module trace status */
+#ifdef __cplusplus
+[[maybe_unused]]
```

Comment:
I don't particularly like this one, but I haven't managed to find a cleaner solution. This is because a static global in a header is not great to begin with. It's practically begging for an unused\-variable error to happen